### PR TITLE
LM box upgrade

### PR DESCRIPTION
**Pull request by adambrown1**
1) Fixes the divide by zero issues and
2) Allows the remaining LMs to be read out even if one doesn't work

### CODE REVIEW

File: n2_lmbox_lan.py [hunk 1]
```diff
@@ -72,9 +81,10 @@ def send_recv(self, message):
             self.logger.fatal("Could not send message %s. Error: %s" % (message.strip(), e))
             ret['retcode'] = -2
             return ret
-        time.sleep(1) # Giving the device more time to respond than normal LAN device
+        time.sleep(2) # Giving the device more time to respond than normal LAN device
         try:
             ret['data'] = self._device.recv(1024)
+            self.logger.debug(f'data: {ret["data"]}')
```
Comment (jarongrigat) [inline n2_lmbox_lan.py]: I would get rid of this, since it is not really easy to interpret and will swamp the log files

**Review by jarongrigat — APPROVED** [review]

File: n2_lmbox_lan.py (hunk 1 above)
Comment (adambrown1) [reply n2_lmbox_lan.py]: Gone in the latest update

**Comment by adambrown1** [comment]
I have significantly changed this - could you look again. It is a lot cleaner now I hope!

**Review by jarongrigat — APPROVED** [review]
Looks great. Can the  hardcoded [capitance]>350 be replaced by something else? Or where does this number come from?

**Comment by adambrown1** [comment]
> Looks great. Can the hardcoded [capitance]>350 be replaced by something else? Or where does this number come from?

It can in principle. It is higher than any capactiance we currently expect, and I want to investigate where these strange readings come from by first writing some more log messages. Maybe I should take it out before merging

**Pull request merged**